lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/jdo/NameValidation.java

```diff
@@ -34,6 +36,9 @@ public static String validateName(String key) {
 		if ("".equals(key)) {
 			throw new IllegalArgumentException("Name cannot be only whitespace or empty string");
 		}
+		if (key.length() > TableConstants.MAX_COLUMN_NAME_SIZE_CHARS) {
```

Comment:
This is the wrong constant.  Add the constant to this class.  We will need tests for the other two cases.